<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Meerkat::SaveSessionDialog</class>
 <widget class="QDialog" name="Meerkat::SaveSessionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Session</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Session title:</string>
       </property>
       <property name="buddy">
        <cstring>titleLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="identifierLabel">
       <property name="text">
        <string>Session identifier:</string>
       </property>
       <property name="buddy">
        <cstring>identifierLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="identifierLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="onlyCurrentWindowCheckBox">
     <property name="text">
      <string>Store only current window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
